content/docs/user-guide/data-management/remote-storage/index.md

```diff
@@ -92,27 +92,29 @@ team.
 
 ## Supported storage types
 
-<admon type="tip">
-
-Guides for each storage type are in progress. For storage types that do not link
-to a specific guide, see
-[`dvc remote add`](/doc/command-reference/remote/add#supported-storage-types)
-and
-[`dvc remote modify`](/doc/command-reference/remote/modify#supported-storage-types).
-
-</admon>
-
 ### Cloud providers
 
 - [Amazon S3] (AWS) and [S3-compatible] e.g. MinIO
-- Microsoft Azure Blob Storage
+- Microsoft [Azure Blob Storage]
 - Google Drive
-- Google Cloud Storage (GCP)
+- [Google Cloud Storage] (GCP)
 - Aliyun OSS
 
+<admon type="tip">
+
+Guides for each storage type are in progress. For storage types that do not link
+to a specific guide, see the
+[`dvc remote modify` reference](/doc/command-reference/remote/modify#supported-storage-types).
+
+</admon>
```

Comment:
Oops didn't mean to move this under the list. Somehow happened during Git merge.

But how about we remove this admon and link every entry to the corresponding `remote modify` section instead for now?

Reply:
> p.s. moved the admon up again for now.

Reply:
TBH I don't have a strong opinion. If you prefer that, feel free to update to that format.

Reply:
Well, for now it's not easy for me to make more changes here since I can't push to the branch anymore but I could open a separate PR from my fork later. Let's keep as-is in this one for now.

Reply:
@jorgeorpinel hey, I invited you back. sorry about that, was not intentional, I don't think we need complicating things here.

Reply:
No worries I assumed it had to do with compliance. I don't mind either way. Thanks for the invite back BTW I was auto-subscribed to 165 repos heh. Any chance I can only be added to this repo instead of the teams to un-do that?

Reply:
yes, sorry about that. I can do that. CML and DVC I guess - just in case.

Reply:
done, you should have the maintainer access to a few repos now

Reply:
Thanks @shcheklein ! OK I linked all the types now. PTAL @dberenbaum 🙂